gdrdrv/gdrdrv.c

```diff
@@ -642,7 +643,7 @@ static int gdrdrv_mmap(struct file *filp, struct vm_area_struct *vma)
     gdr_mr_t *mr = NULL;
     u64 offset;
     int p = 0;
-    unsigned long vaddr, prev_page_paddr;
+    unsigned long vaddr;
     int phys_contiguous = 1;
```

Comment:
is phys_contiguous required? I don't see it used meaningfully anywhere now.

Reply:
right, not anymore